job_templates/ansible_roles_-_ansible_default.erb

```diff
@@ -0,0 +1,20 @@
+<%#
+name: Ansible Roles - Ansible Default
+job_category: Ansible Playbook
+description_format: Run Ansible roles
+snippet: false
+provider_type: Ansible
+kind: job_template
+model: JobTemplate
+%>
+
+---
+- hosts: all
+  tasks:
+    - name: Display all parameters known for the Foreman host
+      debug:
+        var: foreman_params
+  roles:
+<% if @host.all_ansible_roles.present? -%>
```

Comment:
if you add - in opening erb tag, you'll get same lenght of erb tag and have code aligned to even number of whitespaces and then also the line @host.all_ansible ... can be properly aligned

```erb
<%- if @host.all_ansible_roles.present? -%>
<%=   @host.all_ansible_roles.map { |role| "    - #{role.name.strip}" }.join("\n") %>
<%- end %>
```